api4/system.go

```diff
@@ -64,6 +68,150 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.ApiRoot.Handle("/warn_metrics/trial-license-ack/{warn_metric_id:[A-Za-z0-9-_]+}", api.ApiHandler(requestTrialLicenseAndAckWarnMetric)).Methods("POST")
 	api.BaseRoutes.System.Handle("/notices/{team_id:[A-Za-z0-9]+}", api.ApiSessionRequired(getProductNotices)).Methods("GET")
 	api.BaseRoutes.System.Handle("/notices/view", api.ApiSessionRequired(updateViewedProductNotices)).Methods("PUT")
+
+	api.BaseRoutes.System.Handle("/generate_support_packet", api.ApiSessionRequired(generateSupportPacket)).Methods("GET")
```

Comment:
Can we drop "generate" on the RestAPI.  Essentially it is a "Get" "support_packet". I'm ok with `generateSupportPacket` for internal function name.

Reply:
Okay! Done. Thanks :)